ReleaseNotes-2.7

```diff
@@ -3,63 +3,126 @@
- * Improved scheduler speed
- * New waitservice and mergeservice commands to handle _service files
- * Allow non-obs urls in kiwi, map to obs urls if possible
- * collax package format support
- * Support virt-builder repository meta data
+read also the README.UPDATERS file about the necessary steps. Please note that
+you need to have at least OBS 2.5 to be able to update to OBS 2.7 release.
+Migrations from older OBS versions will fail.
+
+OBS Appliance users who have setup their LVM according to our documentation can
+just replace their appliance image without data loss. The migration will happen
+automatically.
+
+OBS 2.7 will be the last release supporting SUSE Linux Enterprise Server 11 as
+operating system for the server installation.
+
+Backend Features
+================
+* make the new full tree handling the default
+* Incremental storage of sources from upstream SCM systems
+  * cpio archives are stored incremental and offered the to the build job
+    as directory.
+  * The default git handling has been changed to use obs_scm service which
+    produces the right format.
+* Download on demand repositories support changing source repos now
+* Source services can be used during build time
+  * they get executed before the packaging tool (eg. rpm-build) gets started
+* Support to use docker as virtualization engine
+* Support for simple image creation
+* New hardware architectures are supported: k1om, aarch64_ilp32
+* Many improvements in the maintenance support features
+* Improved scheduler speed
+* Allow non-obs urls in kiwi, map to obs urls if possible
+* collax package format support
+* Support virt-builder repository meta data
+* bs_admin:
+  * give the admin a chance to prefer publish events with --prefer-publish-event
+  * --vm-enable-console
+  * --rebuild-full-tree
+* Add sse3 and ssse3 to list of valid cpu flags
+* auto-update configuration from time to time
+* add support for peer fingerprint verification
+  Can be used for DoD, interconnect, etc...
+* Basic support for simpleimage
+  Adding support for squashfs and tar.gz binaries and for simpleimage
+  build type (request for build update to support it sent separately)
+* support buildtime source services
+  * skip them on service server
+  * add dependencies to build env
+
+
+Frontend Features
+=================
+* Support embargo of projects (#316776)
+* webui has support to add remote repositories
+* Instantiate package api call for projects building service packs
+* Support tracking of github issues
+* New option to avoid usage of defined repositories when branching a package
+  via the OBS:BranchSkipRepositories attribute
+* Groups can have an email adress for notifications and contact
+* New waitservice and mergeservice commands to handle _service files
+
+* make request numbering reliable when using a MySQL cluster
+* Lots of data input validations
+* use new obs_scm service for git source URLs
+* Make it possible to force Package.destroy
```

Comment:
It is a webui feature. We should seperate webui and api features here again so one gets the context.